multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -35,6 +35,12 @@ type MemberCluster struct {
 
 // ClusterSetSpec defines the desired state of ClusterSet.
 type ClusterSetSpec struct {
+	// Identify the local cluster.
+	// +kubebuilder:validation:Required
+	ClusterID string `json:"clusterID,omitempty"`
+	// Identify the local cluster's membership in a given ClusterSet.
+	// +kubebuilder:validation:Required
+	ClusterSetID string `json:"clusterSetID,omitempty"`
 	// Members include member clusters known to the leader clusters.
 	// Used in leader cluster.
 	Members []MemberCluster `json:"members,omitempty"`
```

Comment:
Question about MemberCluster:

1.  `Server` is used only for leader cluster?
2. `Secret` - the comment says: "Secret name to access API server of the member from the leader cluster". But should not it be the Secret to access the leader cluster by a member cluster? Is it used in member or leader or both?
3. `ServiceAccount` - the field is needed only in leader cluster? These names are not clear to understand (but maybe we want not to rename in this PR).
4. Today we assume users manually (can be through antctl) create a SA and a Secret in the leader cluster for each member? Is that possible to auto-create them by MC Controller?

Reply:
1. The `Server` is used only for member cluster.
2. The comment of `Secret` is incorrect, it stores the token associated with a ServiceAccount in the leader cluster. It's used by member cluster to access the leader's API server. I will refine the comment.
3. Yes, `ServiceAccount` field is needed only in leader cluster. But I didn't see it's been used in codes. It seems only helpful for leader cluster admin to understand which ServiceAccount is associated with the member cluster.
4. A default `ServiceAccount` antrea-mc-member-access-sa will be created when MC manifests are applied in leader cluster, No Secret is created automatically in leader cluster at this moment.  I think we can add a Secret in the leader manifest, so user can reuse the default one if no extra security concern. I will check this.

Reply:
Could you correct/update the comments on the fields?

Reply:
Comments updated.